src/components/App/App.tsx

```diff
@@ -55,7 +55,7 @@ function App() {
       apiCalls.fetchCountriesData()
```

Comment:
- What is the purpose of the fragment (`<>`) that wraps the `form` element?
- Remove console logs before pushing code into the main branch
-  The "DONE"/"TO DO" list in your `App.js` file is more appropriate to put in your PR, project board, and/or issues. Keep that out of your code.